azuki-core/build.gradle

```diff
@@ -6,6 +6,7 @@ plugins {
 dependencies {
     implementation 'org.jetbrains.kotlin:kotlin-test-junit'
     implementation 'com.fasterxml.jackson.module:jackson-module-kotlin:2.12.5'
+    implementation 'com.vdurmont:semver4j:3.1.0'
```

Comment:
⚠️ This package is no longer maintained, but there is an active fork: https://github.com/semver4j/semver4j